homeassistant/components/rss_feed_template.py

```diff
@@ -0,0 +1,100 @@
+"""
+Exports sensor values via RSS feed.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/rss_feed_template/
+"""
+
+import asyncio
+from html import escape
+from aiohttp import web
+
+import voluptuous as vol
+
+from homeassistant.components.http import HomeAssistantView
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = "rss_feed_template"
+DEPENDENCIES = ['http']
+
+CONTENT_TYPE_XML = "text/xml"
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: cv.ordered_dict(
+        vol.Schema({
+            vol.Optional('title'): cv.template,
+            vol.Required('items'): vol.All(
+                cv.ensure_list,
+                [{
+                    vol.Optional('title'): cv.template,
+                    vol.Optional('description'): cv.template,
+                }]
+                )
+            })
+        )
+    }, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Setup the RSS feeds."""
+    for (feeduri, feedconfig) in config[DOMAIN].items():
+        url = '/api/rss_template/%s' % feeduri
+
+        title = feedconfig.get('title')
+        if title is not None:
+            title.hass = hass
+
+        items = feedconfig.get('items')
+        for item in items:
+            if 'title' in item:
+                item['title'].hass = hass
+            if 'description' in item:
+                item['description'].hass = hass
+
+        rss_view = RssView(url, title, items)
+        hass.http.register_view(rss_view)
+
+    return True
+
+
+class RssView(HomeAssistantView):
+    """Export states and other values as RSS."""
+
+    requires_auth = False
```

Comment:
Done. Please note that there is also an update to the (already approved) PR to the doc regarding this parameter.